CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ target_include_directories(${PROJECT_NAME} PUBLIC
     ${PROJECT_SOURCE_DIR}/src
 )
 
+include_directories(${PROJECT_SOURCE_DIR})
+include_directories(${PROJECT_SOURCE_DIR}/src)
+include_directories(${PROJECT_SOURCE_DIR}/src/Core)
```

Comment:
Including ${PROJECT_SOURCE_DIR}/src should be enough. There are no header files outside of it.